ios/swift/AmpliSwiftSampleApp/Shared/ampli.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
I think we should move `ampli.json` to the root dir (same as Podfile)

Reply:
Podfile is in root directory under app now. Is the ampli.json path set in cli?

Reply:
updated